html/pfappserver/lib/pfappserver/Form/Config/Source/HTTP.pm

```diff
@@ -73,6 +73,43 @@ has_field 'authorization_url' =>
    tags => { after_element => \&help,
              help => 'Note : The URL is always prefixed by a slash (/)' },
   );
+has_field 'stripped_user_name' =>
+  (
+   type            => 'Toggle',
+   checkbox_value  => 'yes',
+   unchecked_value => 'no',
+   default         => 'yes',
+   label           => 'Use stripped username ',
+   tags => { after_element => \&help,
+             help => 'Use stripped username returned by RADIUS to test the following rules.' },
+  );
+
+has_field 'realm' =>
```

Comment:
That common field to all sources should be declared in pfappserver::Form::Config::Source if they don't have anything different which seems like the case now